core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGeneData.java

```diff
@@ -114,14 +132,28 @@ public static void importData(File geneFile) throws IOException, DaoException {
             }
         }
         
-        MySQLbulkLoader.bulkLoadOn();
         DaoGeneOptimized daoGene = DaoGeneOptimized.getInstance();
 
         // Add genes with symbol from nomenclature authority
         for (Map.Entry<String, Set<CanonicalGene>> entry : genesWithSymbolFromNomenClatureAuthority.entrySet()) {
             Set<CanonicalGene> genes = entry.getValue();
             if (genes.size()==1) {
-                daoGene.addGene(genes.iterator().next());
+                CanonicalGene gene = genes.iterator().next();
+                //if exists
+                CanonicalGene dbGene = daoGene.getGene(gene.getEntrezGeneId());
+                if (dbGene != null) {
```

Comment:
since we will be only adding genes and not updating them, this check can be removed and only the `else` can stay